### ISSUE TITLE
Remove tag bulk operation unexpected behaviour

### ISSUE THREAD

**Issue by gingerbeardman** [sissbruecker/linkding#203]
I have a bunch of bookmarks with the tag:
#toread

- I clicked on that tag to see them all
- I entered bulk edit mode
- the search field showed me `#toread` but bulk editing to remove this had no effect
- instead, I had to bulk edit remove `toread` without the hash character

**Comment by sissbruecker** [comment]
Yeah, the two different tag notations are a little confusing. The tag field should just strip the hash character.
